Gems/Atom/Tools/AtomToolsFramework/Code/Include/AtomToolsFramework/Util/Util.h

```diff
@@ -290,8 +290,8 @@ namespace AtomToolsFramework
     // Visits all scan folders asynchronously, gathering all of the paths matching the filter. 
     AZStd::vector<AZStd::string> GetPathsInSourceFoldersMatchingFilter(const AZStd::function<bool(const AZStd::string&)> filterFn);
 
-    //! Collect a set of file paths from all project safe folders matching a wild card
-    AZStd::vector<AZStd::string> GetPathsInSourceFoldersMatchingWildcard(const AZStd::string& wildcard);
+    //! Collect a set of file paths from all project safe folders matching an extension
+    AZStd::vector<AZStd::string> GetPathsInSourceFoldersMatchingExtension(const AZStd::string& extension);
```

Comment:
Could you bump the `AtomToolsFramework/gem.json` `version` to `1.0.0` if this is a breaking API change?    More info here: https://github.com/o3de/community/blob/main/guides/o3de-version-guidelines.md